readme.txt

```diff
@@ -161,6 +161,9 @@ Stop by our [support site](https://www.mailpoet.com/support).
 
 == Changelog ==
 
+= 3.11.2 - 2018-10-09 =
+* Added: Linux cron option for sending emails.
```

Comment:
Repull the changelog, I added entries for a few other tickets that got missed, and this looks great to me.